pywbem_mock/_wbemconnection_mock.py

```diff
@@ -523,7 +523,9 @@ def compile_mof_file(self, mof_file, namespace=None, search_paths=None,
             The name of the target CIM namespace in the CIM repository. This
             namespace is also used for lookup of any existing or dependent
             CIM objects. If `None`, the default namespace of the connection is
-            used.
+            used.  A namespace defined in a namespace pragma of the MOF
+            has precedence over the namespace parameter.  The namespace
+            specified in this parameter or the MOF itself must exist.
```

Comment:
The notion of "precedence" does not sufficiently explain it. See my previous comment.